.github/workflows/ui-tests-android-sample.yml

```diff
@@ -0,0 +1,19 @@
+name: Unit tests of Android Sample
+
+on:
+  push:
+    branches:
+      - develop
```

Comment:
change to:
```
on:
  pull_request:
    branches:
    - master
    - develop
```
to autocheck all requests without merge to develop

Reply:
Fixed.